target-platform/p2-repo-common/pom.xml

```diff
@@ -172,6 +172,11 @@
 									<version>${org.apache.commons.fileupload.version}</version>
 								</artifactItem>
 								<artifactItem>
+								    <groupId>org.apache.commons</groupId>
+								    <artifactId>commons-exec</artifactId>
+								    <version>${commons-exec.version}</version>
```

Comment:
The build script can't handle this right now. The final name will be converted by P2 from 1.3 to 1.3.0, so the file name will be first 1.3 and then 1.3.0. I can hardcode this, but this seems even more wrong to me.